Updating GeoFence documentation with example on CQL read filter limits on time range (#5130)
geoserver/geoserver: doc/en/user/source/extensions/geofence-server/gui.rst

Currently, IPv6 style address ranges are not supported by GeoFence.

When Access type LIMIT is selected, additional options are displayed that allows the user to select the Catalog Mode and the Allowed Area (WKT) associated with this rule. The Spatial Filter Type parameter allows to define whether apply the Allowed Area filter to vector data as an Intersects or a Clip filter. 

.. figure:: images/limit.png
   :align: center
  
When Access type ACCESS is selected as well as a specific layer, it becomes possible to specify the "layer details" in a separate tab.
These make it possible to add additional filters to the layer for the rule in question. For example, the rule can alter the default style of the layer, specify which styles are available at all, which attributes are accessible for reading and/or writing, specify CQL filters for both reading and writing, specify a catalog mode, and an allowed area (WKT) filter.

.. figure:: images/layerdetails.png
   :align: center

A CQL Read Filter can be setup to allow users to only access a subset of the available data.

> For example, if you have a daily time series layer (a Layer with time attribute/dimension), 
> you can set a rule allowing users to only access data in a certain time range.

Reviewer: ```suggestion
For example, having a daily time series layer (a Layer with time attribute/dimension), 
it's possible to restrict access to a given time range.
```

> A CQL filter to limit access to the whole 2020 year (static time range) will be like this::

Reviewer: ```suggestion
A CQL filter to limit access to the whole 2020 year (static time range) would look as follows::
```

   time between 2020-01-01 and 2020-12-31

> A CQL filter to only access data older than a week (dynamic time range) will be like this::

Reviewer: ```suggestion
A CQL filter limiting access to data older than a week (dynamic time range) would look like this instead:
```
